### DISCUSSION TITLE
My graphql is suddenly failing

### DISCUSSION THREAD

**Discussion by tombrus** [community/community#24522] [Apps, API and Webhooks]
<p>During one of my actions I used to get packages in my repo with something like (I left out the exact query for brevity):</p>
<pre><code class="lang-auto">query { 
  repository(owner:"ModelingValueGroup", name:"buildTools") {
    registryPackages
  }
}
</code></pre>
<p>but now this code suddenly fails with</p>
<pre><code class="lang-auto">Field 'registryPackages' doesn't exist on type 'Repository'
</code></pre>
<p>the API docs say it exists: <a href="https://developer.github.com/v4/object/repository/" rel="nofollow noopener">https://developer.github.com/v4/object/repository/</a></p>
<p>What happened?</p>

**Comment by tombrus** [comment, marked as answer]
<p><em>Self-answer</em>:<br>
I guess some renaming was done in the API recently: <code>registryPackages</code> seems to be <code>packages</code> now.<br>
For more details see <a href="https://github.community/t/how-to-list-packages-and-versions-in-github-package-registry/16999/3" class="inline-onebox">How to list packages and versions in github package registry?</a></p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for coming back to share your solution <a class="mention" href="/tombrus">@tombrus</a>, it’ll help someone!</p>
